### PR TITLE
Connected PostgreSQL

### PR DESCRIPTION
**Pull request by andriiavdiiuk**
Connected PostgreSQL.
Added Entities.

### CODE REVIEW

File: GotExplorer.API/appsettings.json [hunk 1]
```diff
@@ -10,5 +10,9 @@
         "Key": "YourSecretKeyForAuthenticationOfApplication",
         "Issuer": "Imbulii",
         "Audience": "Imbulii"
-    }
+    },
+    "ConnectionStrings": {
+        "DefaultConnection": "Server=localhost;Port=5432;Database=GotExplorer;Username=postgres;Password=1111"
```
Comment (Evgeniy1996) [inline GotExplorer.API/appsettings.json:15]: Let's use .net secrets

File: GotExplorer.API/Program.cs [hunk 2]
```diff
@@ -141,12 +145,28 @@ public static void Main(string[] args)
...
+        {
+            var roles = serviceProvider.GetRequiredService<IConfiguration>().GetSection("Roles").Get<string[]>();
+            var roleManager = serviceProvider.GetRequiredService<RoleManager<UserRole>>();
+            foreach (var role in roles)
+            {
+                if (!await roleManager.RoleExistsAsync(role))
+                {
+                    await roleManager.CreateAsync(new UserRole(role));
+                }
+            }
```
Comment (Evgeniy1996) [inline GotExplorer.API/Program.cs:169]: Is not async actions exist ? 
There is not reason to use async method and wait action below

File: GotExplorer.DAL/Entities/User.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Identity;
+using GotExplorer.DAL.Interfaces;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations.Schema;
+namespace GotExplorer.DAL.Entities
+{
+    public class User : IdentityUser<int>, IEntity<int>
+    {
+        [ForeignKey("Image")]
+        public int ImageId { get; set; } = 1;
```
Comment (Evgeniy1996) [inline GotExplorer.DAL/Entities/User.cs:10]: We should not use predefined value equal to 1

File: GotExplorer.DAL/Entities/Game.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+{
+    public class Game : IEntity<int>
+    {
+        public int Id { get; set; }
+
+        public int Score { get; set; }
+
+        public TimeSpan SpentTime { get; set; }
+
+        [ForeignKey("User")]
```
Comment (Evgeniy1996) [inline GotExplorer.DAL/Entities/Game.cs:15]: I would recommend to use this attribute  on  the navigation property. 
I would recommend to use this approach or every where or never

File: GotExplorer.API/Program.cs [hunk 5]
```diff
@@ -4,14 +4,15 @@
...
 using Microsoft.IdentityModel.Tokens;
 using System.IdentityModel.Tokens.Jwt;
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using GotExplorer.DAL;
 using Microsoft.EntityFrameworkCore;
-using GotExplorer.DAL.Models;
+using GotExplorer.DAL.Entities;
+using GotExplorer.BLL.Services.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
 namespace GotExplorer.API
```
Comment (Evgeniy1996) [inline GotExplorer.API/Program.cs:16]: empty line

**Review by Evgeniy1996 — APPROVED** [review]

**Review by wowdiamon — APPROVED** [review]

**Review by infalliblePirate — APPROVED** [review]

**Review by uliaescha — APPROVED** [review]

**Pull request merged**